experiments/examples/README.md

```diff
@@ -0,0 +1,40 @@
+# Experiments CLI
+In this directory we have a command line interface that helps us interact with the Nimbus SDK.
+
+
+## How to use
+In order to use the CLI, you would run it as an example:
+```bash
+cargo run --example experiment -- -c ./examples/context.json show_all
+```
+This would should display all the valid experiments retrieved from the server
+
+There are a bunch of options, which you can find if you do:
+```bash
+cargo run --example experiment -- -h
+```
+Which will show you the following:
+```
+Nimbus SDK 
+Tarik E. <teshaq@mozilla.com>
+A demo for the Nimbus SDK
+
+USAGE:
+    experiment [OPTIONS] --context <FILE> [SUBCOMMAND]
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -b, --bucket_name <String>        Sets a custom bucket name, defaults to `main`
+    -l, --collection_name <String>    Sets a custom collection name, defaults to `messaging-experiments`
+    -c, --context <FILE>              Sets a custom Application Context
+    -s, --server <URL>                Sets a custom remote settings server url
+    -u, --uuid <UUID>                 Sets a custom uuid to be used for bucketing and branch selection, defaults to a randomly generated uuid
```

Comment:
as discussed in Slack, let's throw these into a JSON file as well, 
and you can define your own schema for it, such as:

e.g.
```diff
{
server: ....,
collection: ...,
uuid: ...,
appContext: {
    ...
  }
}
```